src/components/ShareToken/ShareToken.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import { ReactComponent as ShareIcon } from '../../images/icon-share.svg';
+import { ReactComponent as CopyIcon } from '../../images/icon-copy.svg';
```

Comment:
This is a nice feature! I feel like I learn something on every PR lately.

Reply:
I just learned you could do it this way! Super handy, and let's you micromanage what's going on with the style of the different parts of the SVG if you put classes on the different shapes.